lib/OpenQA/WebAPI/Controller/Main.pm

```diff
@@ -140,8 +140,17 @@ sub group_overview {
     return $self->reply->not_found unless $group;
 
     my $res = $self->_group_result($group, $limit_builds);
-    my @comments = $group->comments->all;
-    for my $comment (@comments) {
+    my @comments;
+    my @pinned_comments;
+    for my $comment ($group->comments->all) {
+        # find pinned comments
+        if ($comment->user->is_operator && CORE::index($comment->text, 'pinned-description') >= 0) {
```

Comment:
The behaviour as you describe it doesn't make any sense. However, the code is supposed to behave different:
Everybody can create comment with 'pinned-comment' keyword, indeed. But only if the _commentator_ is an operator (or an admin) the comment will be actually treated as such. Any user (regardless whether operator or not) can see pinned comments.

Note that the code tests whether ```$comment->user->is_operator``` and not whether ```current_user->is_operator```.